leif/parse_slo_rules.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package leif
+
+import (
+	"encoding/json"
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var oneSec, _ = time.ParseDuration("1s")
```

Comment:
You should not need to manually instantiate this variable, the `time` package supports several [constants](https://golang.org/pkg/time/#pkg-constants) to support this e.g. `time.Second`, `time.Minute`, `time.Nanosecond` etc.

Reply:
I don't know why I didn't check that 🤦 Thank you!!